rmw_connextdds_common/src/common/rmw_impl.cpp

```diff
@@ -922,6 +924,31 @@ RMW_Connext_Publisher::assert_liveliness()
   return RMW_RET_OK;
 }
 
+rmw_ret_t
+RMW_Connext_Publisher::wait_for_all_acked(rmw_time_t wait_timeout)
+{
+  DDS_Duration_t timeout;
```

Comment:
I would assume infinite timeout by default, and then change the `if()` below to check for `wait_timeout != RMW_DURATION_INFINITE`

```suggestion
  DDS_Duration_t timeout = DDS_DURATION_INFINITE;
```